tests/utilities/test_data.py

```diff
@@ -112,3 +113,29 @@ def test_has_len_all_rank():
         assert not has_len_all_ranks(DataLoader(RandomDataset(0, 0)), trainer.training_type_plugin, model)
 
     assert has_len_all_ranks(DataLoader(RandomDataset(1, 1)), trainer.training_type_plugin, model)
+
+
+def test_replace_dataloader_init_method():
```

Comment:
Why not remove the test of the same name in `tests/lite/test_lite.py`?

This test here is a better unit-test for the functionality.

Additionally, no tests were added to Lite for functionality already existing in Lightning (accelerator connector parsing for example).

Tests should try to be orthogonal, and these 2 are not.

Reply:
I simplified the test, but the test has to stay I'm pretty sure.